### PR TITLE
[   MAISTRA-749] Reverted grafana patch to pick up settings

### PR DESCRIPTION
**Pull request by brian-avery**
This pr adds grafana_patch_values back in. The grafana configmap is parsed as a string and is not easily mergeable. As such, our options are to include this in patch-grafana.sh for now or to include everything in the base SMCP. This also fixes the unit tests for operator.

### CODE REVIEW

**Review by jwendell — APPROVED** [review]

File: pkg/controller/servicemesh/controlplane/controller_test.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func TestCyclicTemplate(t *testing.T) {
 	reconciler := reconcileControlPlane.getOrCreateReconciler(&v1.ServiceMeshControlPlane{})
 	reconciler.Log = log.WithValues()
 
-	_, err := reconciler.renderSMCPTemplates(v1.ControlPlaneSpec{Template: "visited"}, map[string]struct{}{"visited": {}})
+	_, err := reconciler.recursivelyApplyTemplates(v1.ControlPlaneSpec{Template: "visited"}, map[string]struct{}{"visited": {}})
```
Comment (knrc) [inline pkg/controller/servicemesh/controlplane/controller_test.go]: @brian-avery Was this change required by the grafana work?

**Review by knrc — CHANGES_REQUESTED** [review]

File: pkg/controller/servicemesh/controlplane/controller_test.go (hunk 1 above)
Comment (brian-avery) [reply pkg/controller/servicemesh/controlplane/controller_test.go]: @knrc No, this change wasn't required by the Grafana work. If you'd like I'll move it to another PR. When going through CR comments we changed method names and this changes the method names in the unit tests to match the actual function name.

Comment (brian-avery) [reply pkg/controller/servicemesh/controlplane/controller_test.go]: We don't run unit tests so there's nothing that actually cares if this change is in there.

Comment (knrc) [reply pkg/controller/servicemesh/controlplane/controller_test.go]: Yes please, we should move this to another PR

Comment (brian-avery) [reply pkg/controller/servicemesh/controlplane/controller_test.go]: Removed

Comment (brian-avery) [reply pkg/controller/servicemesh/controlplane/controller_test.go]: this change is now in: https://github.com/Maistra/istio-operator/pull/214

**Review by knrc — APPROVED** [review]

**Comment by jwendell** [comment]
Please folks, squash the commits before merging
